addons/placement/functions/fnc_updatePreview.sqf

```diff
@@ -0,0 +1,62 @@
+#include "script_component.hpp"
+
+private _pos = AGLtoASL screenToWorld getMousePosition;
+private _class = call FUNC(getSelectedClass);
+
+#define OFFSET 10
+
+if (_class isEqualTo "") exitWith {
+	if !(GVAR(previewVehicle) isEqualTo objNull) then {
+		deleteVehicle GVAR(previewVehicle);
+		deleteVehicle GVAR(previewObject);
+		GVAR(previewVehicle) = objNull;
+	};
+};
+
+// Delete the old preview if the player selects a different item
+if !(_class isEqualTo (typeOf GVAR(previewVehicle))) then {
+	deleteVehicle GVAR(previewVehicle);
+	deleteVehicle GVAR(previewObject);
+	GVAR(previewVehicle) = objNull;
```

Comment:
Indentation trouble. Ia there no EditorConfig in this repo yet? Someone should add it!

Reply:
Editor config is present but I guess their editors don't support it or it's disabled.